src/Workspaces/Core/Portable/Workspace/WorkspaceDiagnosticDescriptors.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    internal sealed class WorkspaceDiagnosticDescriptors
+    {
+        internal readonly static DiagnosticDescriptor ErrorReadingFileContent;
+
+        internal const string ErrorReadingFileContentId = "IDE1100";
+
+        static WorkspaceDiagnosticDescriptors()
+        {
+            ErrorReadingFileContent = new DiagnosticDescriptor(
+                id: ErrorReadingFileContentId,
+                title: new LocalizableResourceString(nameof(WorkspacesResources.Workspace_error), WorkspacesResources.ResourceManager, typeof(WorkspacesResources)),
+                messageFormat: new LocalizableResourceString(nameof(WorkspacesResources.Error_reading_source_file_content), WorkspacesResources.ResourceManager, typeof(WorkspacesResources)),
+                category: WorkspacesResources.Workspace_error,
```

Comment:
category should not be localizable string, though it seems we are using localizable strings for it elsewhere too. I will fix this in a separate PR.

Reply:
OK, thanks!